src/NovelRT.Interop/Graphics/NrtFontSet.cpp

```diff
@@ -0,0 +1,55 @@
+
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#include "NovelRT.h"
+#include "../NrtInteropErrorHandlingInternal.h"
+#include "NovelRT.Interop/NrtInteropUtils.h"
+#include "NovelRT.Interop/Graphics/NrtFontSet.h"
```

Comment:
roses are red, violets are blue, these are local includes, shouldn't they be system too? :^)